NEWS.md

```diff
@@ -1,5 +1,5 @@
-## Version 0.13.2 (yyyy-mm-dd)
-
+## Version 0.13.2 (2019-09-10)
```

Comment:
What a nice thing still being in 0.x!! We don't need to release a major version!